src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Lottos {
+    public static final int LOTTO_PRICE = 1000;
+    public static final int WINNING_STANDARD_NUMBER = 3;
+    private final List<Lotto> lottoList;
+    private final int lottoCount;
+
+    public Lottos(List<Lotto> lottoList, int lottoCount) {
+        this.lottoList = lottoList;
+        this.lottoCount = lottoCount;
+    }
+
+    public static Lottos of(int lottoPrice) {
+        int count = lottoPrice / LOTTO_PRICE;
+        return new Lottos(
+                Stream.generate(() -> Lotto.createLotto())
```

Comment:
메서드 레퍼런스를 사용해볼 수 있겠네요! 😄 
```suggestion
                Stream.generate(Lotto::createLotto)
```

Reply:
더 간편하게 표현이 가능했군요. 다음과 같이 변경하였습니다. ^^

Reply:
로또를 랜덤으로 생성하는 과정에서 createLotto에 매개변수가 생겨 다시 수정했습니다!